Adding flax nodes from 1416

msmolev opened the pull request:
Adding all nodes from https://github.com/gw2efficiency/issues/issues/1416

Images for these entries:

[node_images.zip](https://github.com/gw2efficiency/static-data/files/6288903/node_images.zip)

queicherius commented:
Looks good! The only thing the pipeline is complaining about is the missing newline at the end of the file.

> (not sure what is "optimal" field for, presume for ordering so moved it out with higher IDs)

Yep, this is for ordering when set to "optimal", you can set those to something that would make sense in order (e.g. the ones in the same zone with numbers that follow each other)

msmolev commented:
- Added elder wood nodes, orichalcum node from initial issue (only one, as author noted the other one is not permanent)
- Added images into zip archive for wood and orichalcum node (resized to 690px width where applicable)
- Fixed linting issues

queicherius commented:
Awesome, thank you! 👍
